src/declarations.d.ts

```diff
@@ -18,7 +18,8 @@ declare module 'cozy-ui/transpiled/react/providers/I18n' {
 
 declare module 'cozy-ui/transpiled/react/deprecated/Alerter' {
   const Alerter: {
-    error: (message: string) => void
+    error: (message: string, options?: Record<string, unknown>) => void
+    success: (message: string, options?: Record<string, unknown>) => void
```

Comment:
Should we modify cozy-ui to expose those new definitions?

Reply:
yes ideally, in this case I took the fastest way, I think it can be done after publication in test flight (for flagship too)